### ISSUE TITLE
Collection Icon problem

### ISSUE THREAD

**Issue by wuchunhuan** [outline/outline#7576]
I noticed a lock was added to one of my collection icon at the bottom right corner in the Collections list, other collections are not affected though. 
![image](https://github.com/user-attachments/assets/2f577fe4-6522-4708-af46-5acdd268f12a)

First I thought it may have something to do with the collection permission, but i never changed it or any other settings, and I have another collection with the same permission(all members: No access, me: Manage), its icon has no lock attached to it.

 I tried to change to other icon, and no lock is attached, and then change back to the original one, lock is back on(the original icon has no lock), so wierd... 

what bothers me is that I dont't know what triggers this icon problem, I think it's bug, the only change I made was upgrading to the latest version: v0.79.1, any idea how to fix this?

_Originally posted by @wuchunhuan in https://github.com/outline/outline/discussions/7573_

**Comment by tommoor** [comment]
The icon was added in `0.79.0` to represent a collection with restricted access, this is by design

**Comment by wuchunhuan** [comment]
> @wuchunhuan does the lock go away if you change to another icon?

yes, it dose, it seems only this icon has a lock, as @tommoor said, this icon "represent a collection with restricted access", so I assume this icon is special.
